### PR TITLE
Bug 1384605 - Create a simple load test script in XCUITest

### PR DESCRIPTION
**Pull request by npark-mozilla**

### CODE REVIEW

**Comment by farhanpatel** [comment]
### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
SiteLoadTest.swift | 30 | if,for,while,do statements shouldn't wrap their conditionals in parentheses. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: XCUITests/SiteLoadTest.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+// XCode Profiler to measure the memory/CPU load
+class SiteLoadTest: BaseTestCase {
+        
+    var navigator: Navigator!
+    var app: XCUIApplication!
+    
+    override func setUp() {
+        super.setUp()
+        app = XCUIApplication()
+        navigator = createScreenGraph(app).navigator(self)    }
```
Comment (farhanpatel) [inline XCUITests/SiteLoadTest.swift]: `}` should be on a new line

**Comment by garvankeeley** [comment]
@npark-mozilla can you close if not needed in the open PR list?

**Comment by npark-mozilla** [comment]
@garvankeeley actually I was going to merge it after the xcode9 migration is done, but perhaps I should close it now and reopen?

**Comment by garvankeeley** [comment]
Up to you, an extra PR in the list is not a big deal, I am just pinging all these open PRs for people to close them if they can. If the list is small we can audit it at a glance for things we are failing to address (we have some contrib PRs that seem to fall-through-the-cracks)

**Comment by npark-mozilla** [comment]
@farhanpatel could you also merge this one as well? thanks!

**Pull request merged**